pkg/cfn/builder/vpc.go

```diff
@@ -32,6 +35,8 @@ func (c *ClusterResourceSet) addSubnets(refRT *gfn.Value, topology api.SubnetTop
 
 		switch topology {
 		case api.SubnetTopologyPrivate:
+			// Choose the appropriate route table for private subnets
+			refRT = gfn.MakeRef(fmt.Sprintf("%v%v", "PrivateRouteTable", strings.ToUpper(strings.Join(strings.Split(az, "-"), ""))))
```

Comment:
Perhaps this could be simplified to: `fmt.Sprintf("PrivateRouteTable%v", strings.ToUpper(strings.ReplaceAll(az, "-", ""))`. What do you think?

(also for lines 329, 363, 394)